### PR TITLE
Documentation: remove unnecessary instructions

### PR DESCRIPTION
**Pull request by dongsupark**
Now that we merged necessary PRs in lokomotive, we can skip several steps in the installation instructions.

### CODE REVIEW

File: Documentation/install.md [hunk 1]
```diff
@@ -13,84 +13,7 @@ From the [releases](https://github.com/kinvolk/inspektor-gadget/releases).
...
-dns_zone = "CHANGEME"
-dns_zone_id = "CHANGEME"
-ssh_pubkey = "~/.ssh/id_rsa.pub"
-```
-
-Install your cluster:
-```
-$ lokoctl cluster install
-```
+Install your cluster following the [Lokomotive docs for AWS](https://github.com/kinvolk/lokomotive-kubernetes/blob/master/docs/flatcar-linux/aws.md).
```
Comment (alban) [inline Documentation/install.md]: we need to add 
```
os_image = "flatcar-edge"
```
somewhere

Comment (alban) [reply Documentation/install.md]: https://github.com/kinvolk/lokomotive-kubernetes#try-flatcar-edge

Comment (dongsupark) [reply Documentation/install.md]: Done. Added a link to the Flatcar Linux edge.

**Comment by alban** [comment]
Thanks!

**Pull request merged**